frontend/libs/studio-components/src/components/StudioAvatar/StudioAvatar.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { type ReactElement } from 'react';
+import classes from './StudioAvatar.module.css';
+import { PersonCircleIcon } from '@studio/icons';
+
+type ImageDetails = {
+  src: string;
+  alt: string;
+  title: string;
+};
+
+export type StudioAvatarProps = {
+  imageDetails?: ImageDetails;
```

Comment:
What do you think of `avatarElement` instead? 😊 
```suggestion
  avatarElement?: AvatarElement;
```